src/app/tree-grid/tree-grid-validator-service-extended/tree-grid-validator-service-extended.component.html

```diff
@@ -3,20 +3,24 @@
         foreignKey="ParentID" [width]="'100%'" [height]="'500px'" [rowEditable]="true"
         (formGroupCreated)="formCreateHandler($event)">
         <igx-column field="ID" header="ID" dataType="number" [editable]="false"></igx-column>
-        <igx-column field="Age" header="Age" dataType="number" minAge="18" [editable]="true" required max="99">
+        <igx-column field="Age" header="Age" dataType="number" min="18" [editable]="true" required max="99">
         </igx-column>
         <igx-column field="Name" header="Full Name" dataType="string" [editable]="true" required>
         </igx-column>
         <igx-column field="Phone" header="Phone" dataType="string" [editable]="true" required phoneFormat="^[^a-zA-Z]*$">
-            <ng-template igxCellValidationError let-cell='cell'>
+            <ng-template igxCellValidationError let-cell='cell' let-defaultErr="defaultErrorTemplate">
+                <ng-container *ngTemplateOutlet="cell.defaultErr">
+                </ng-container>
                 <div *ngIf="cell.errors?.['phoneFormat']">
                     Please enter correct phone format
                 </div>
             </ng-template>
         </igx-column>
         <igx-column field="Title" header="Title" dataType="string" [editable]="true" required></igx-column>
         <igx-column field="HireDate" header="Hire Date" dataType="date" [editable]="true" required>
-            <ng-template igxCellValidationError let-cell="cell">
+            <ng-template igxCellValidationError let-cell="cell" let-defaultErr="defaultErrorTemplate">
+                <ng-container *ngTemplateOutlet="cell.defaultErr" >
```

Comment:
This still does not work, please update the ngTemplateOutlet as well:

<ng-container *ngTemplateOutlet="defaultErr" >